.papr.yml

```diff
@@ -4,128 +4,64 @@ branches:
     - try
 
 required: true
-context: f25-sanitizer
+context: f25-primary
 
 container:
-    image: projectatomic/ostree-tester
-
+    image: registry.fedoraproject.org/fedora:25
 packages:
-  - libasan
   - git
-  - coccinelle
 
 env:
     CFLAGS: '-fsanitize=undefined -fsanitize-undefined-trap-on-error -fsanitize=address -O2 -Wp,-D_FORTIFY_SOURCE=2'
     ASAN_OPTIONS: 'detect_leaks=0'  # Right now we're not fully clean, but this gets us use-after-free etc
     # TODO when we're doing leak checks: G_SLICE: "always-malloc"
 
-build:
-    config-opts: >
-      --prefix=/usr
-      --libdir=/usr/lib64
-      --enable-installed-tests=exclusive
```

Comment:
We're losing this tidbit, so the `g-d-t-r` invocation in `build-check.sh` won't actually run anything right?